crates/core/tedge_agent/src/main.rs

```diff
@@ -4,6 +4,7 @@ use clap::Parser;
 use tedge_config::system_services::get_log_level;
 use tedge_config::system_services::set_log_level;
 use tedge_config::DEFAULT_TEDGE_CONFIG_PATH;
+use tracing::log::warn;
```

Comment:
Is using trace a good idea jut to log a `warning`?

Reply:
I just used tracing as `tedge-agent` is already using the `tracing` crate for logging and I didn't want to add `log` as well as an additonal dependency. May be we should migrate all the crates to use just `log` as we had agreed a while ago, but in a separate PR.